tests/testcrate/tests/ui/sql_table.rs

```diff
@@ -21,15 +21,21 @@
 
 //! Tests of the `#[SqlTable]` attribute.
 
-#![feature(proc_macro)] // FIXME: bad span for field nested_options in stderr (should be on Option<String>, not just Option).
+#![feature(proc_macro_hygiene)] // FIXME: bad span for field nested_options in stderr (should be on Option<String>, not just Option).
```

Comment:
I'm not sure what you're talking about. It looks good in stderr.

Reply:
this particular comment isn't mine, i only fixed feature gate... I will remove it anyway, since span issue seems fixed